yarn.lock

```diff
@@ -0,0 +1,4936 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
please delete this file since this repo no longer uses Yarn (has been replaced by package-lock.json)